Sources/NTPPacket.swift

```diff
@@ -13,9 +13,9 @@ private let kMaximumDispersion = 100.0
 /// - returns: The current time in EPOCH timestamp format.
 func currentTime() -> TimeInterval {
     var current = timeval()
+    /// Thread-safe reference (section 2.9.1): https://pubs.opengroup.org/onlinepubs/9699919799/functions/V2_chap02.html
```

Comment:
Great!
BTW, I couldn't find `gettimeofday` from the section 2.9.1. Do you have more explanation?

Reply:
It seems linked Apple forum's postings have a great explanations.
Please
- Link it too.
- Copy & paste core explanations into the code.